charts/tekton-pipelines/templates/buildpacks/pipelines/_buildpack.yaml

```diff
@@ -125,12 +129,15 @@ spec:
           value: "$(tasks.detect-nodejs-version.results.node-version)"
         - name: environment
           value: "$(params.environment)"
+        - name: docker_registry_repository
```

Comment:
Why do you need this? We already have a `docker_registry` variable here (path to which docker image should be pushed)